packages/api-plugin-fulfillment-method-shipping-flat-rate/README.md

```diff
@@ -0,0 +1,69 @@
+# api-plugin-fulfillment-method-shipping-flat-rate
```

Comment:
We shouldn't include all this boilerplate from the example plugin that describes what's in the example plugin

Reply:
updated